packages/example/src/components/remote-control/RemoteControlManager.ts

```diff
@@ -19,11 +19,15 @@ class RemoteControlManager implements RemoteControlManagerInterface {
       Backspace: SupportedKeys.Back,
```

Comment:
I wish not to use that ugly hack in my code 🤪. What is happening when I pressed the back button on the Lg Magic Remote is that an even is raised with
`event.code ` empty and `event.key === 'GoBack'`
The `mappedKey` is mapped on `event.code` so it doesn't trigger the desired key.

Any idea about how to fix this in a proper way?
🙏

Reply:
Hahaha alright, that's the thing I missed indeed!

Then this will work. It's just the example app anyway.

My suggestion though would be to make the if branch more explicit as an exit condition (following [this](https://medium.com/swlh/return-early-pattern-3d18a41bba8)). What do you think? 😁 

```tsx
    //For LG WebOs Magic Remote
    if (event.key === 'GoBack') {
      this.eventEmitter.emit('keyDown', SupportedKeys.Back);
      return
    }

    if (!mappedKey) {
      return;
    }
    
    this.eventEmitter.emit('keyDown', mappedKey);
```

Reply:
Oh yes! Far better! 
It's a sample app, but it's my favorite TV app around! There are some options for the future we can discuss like:
 - Promoting this sample as a boilerplate for TV!

I have a private repo of this app I use for studying/experimenting/presenting.

Reply:
Do you have a reference for using `.key` on the event? [webOS docs](https://webostv.developer.lge.com/develop/guides/back-button#handling-a-back-button-event) suggest using `keyCode === 461`

Reply:
I've tested on an LG C2 and experience they key is `code` was empty and `key` is GoBack

Reply:
Did you find that `keyCode` was empty as well? If that's the case then their docs are wrong (not the first time) 😅 

I don't have a smart remote available, but testing on simulator (v24) and with LG 32LQ630BPUA `keyCode` behaves as documented

Reply:
Yes keycode was empty using this app. @matthewcaminiti Can you test the app in this repos with your TV/Emulator and check what code is returned?

Reply:
Ah good to know! I gotta test on a newer device. May need to use both `key`/`keyCode` to cover old+new WebOS.

On my TV device (LG 32LQ630BPUA ... old model) I've yet to figure out how to get logs from it, but when using `key === 'GoBack'` it doesn't work, suggesting `key` is empty.

On simulator it's as follows:

https://github.com/bamlab/react-tv-space-navigation/assets/24964496/1a813450-4f09-4578-8819-b63806d10357